pkg/service/test/test.go

```diff
@@ -541,6 +541,8 @@ func (t *tester) FetchTestResults(cfg *FetchTestResultsConfig) models.TestRunSta
 
 	pp.Printf("\n <=========================================> \n  TESTRUN SUMMARY. For testrun with id: %s\n"+"\tTotal tests: %s\n"+"\tTotal test passed: %s\n"+"\tTotal test failed: %s\n <=========================================> \n\n", cfg.TestReport.TestSet, cfg.TestReport.Total, cfg.TestReport.Success, cfg.TestReport.Failure)
 
+	t.logger.Info("Upload your test reports for better visualization at https://beta.keploy.io/test-report")
```

Comment:
This log has to be prinited only 1 time but since it is in the FetchTestStatus this will be logged for each testset. Move this log to the `cmd/test` package at the defer of cmd.Run